src/data/matrix/basic.lean

```diff
@@ -1442,6 +1442,11 @@ lemma conj_transpose_reindex [has_star α] (eₘ : m ≃ l) (eₙ : n ≃ o) (M
   (reindex eₘ eₙ M)ᴴ = (reindex eₙ eₘ Mᴴ) :=
 rfl
 
+lemma mul_transpose_eq_reindex_mul_reindex_transpose [fintype n] [fintype m] [semiring α]
```

Comment:
The statement uses `minor` instead of `reindex`. So I guess the name should be
```suggestion
lemma mul_transpose_eq_minor_mul_minor_transpose [fintype n] [fintype m] [semiring α]
```